spec/api-session-spec.ts

```diff
@@ -895,6 +895,8 @@ describe('session module', () => {
         expect(item.getMimeType()).to.equal('application/pdf');
         expect(item.getReceivedBytes()).to.equal(mockPDF.length);
         expect(item.getTotalBytes()).to.equal(mockPDF.length);
+        expect(item.percentComplete()).to.equal(100)
+        expect(item.currentSpeed()).to.equal(0)
         expect(item.getContentDisposition()).to.equal(contentDisposition);
```

Comment:
not sure how to test `getEndTime()`, but I also didn't see one for `getStartTime()` either.

Reply:
@theogravity I think our best bet here is just to roughly approximate them - something like this should be fine:

```js
diff --git a/spec/api-session-spec.ts b/spec/api-session-spec.ts
index 897ad0ce24..c867117f57 100644
--- a/spec/api-session-spec.ts
+++ b/spec/api-session-spec.ts
@@ -854,6 +854,8 @@ describe('session module', () => {
       });
 
       it('can perform a download with a valid auth header', async () => {
+        const today = Date.now() / 1000;
+
         const server = http.createServer((req, res) => {
           const { authorization } = req.headers;
           if (!authorization || authorization !== 'Basic i-am-an-auth-header') {
@@ -895,9 +897,14 @@ describe('session module', () => {
         expect(item.getMimeType()).to.equal('application/pdf');
         expect(item.getReceivedBytes()).to.equal(mockPDF.length);
         expect(item.getTotalBytes()).to.equal(mockPDF.length);
-        expect(item.percentComplete()).to.equal(100)
-        expect(item.currentSpeed()).to.equal(0)
+        expect(item.getPercentComplete()).to.equal(100)
+        expect(item.getCurrentSpeed()).to.equal(0)
         expect(item.getContentDisposition()).to.equal(contentDisposition);
+
+        const start = item.getStartTime();
+        const end = item.getEndTime();
+        expect(start).to.be.greaterThan(today)
+        expect(end).to.be.greaterThan(start)
       });
 
       it('throws when called with invalid headers', () => {
```